content/tutorials/integration-tutorials/huggingface.md

```diff
@@ -6,79 +6,99 @@ menu:
 title: Hugging Face
 weight: 3
 ---
+
 {{< img src="/images/tutorials/huggingface.png" alt="" >}}
 
 {{< cta-button colabLink="https://colab.research.google.com/github/wandb/examples/blob/master/colabs/huggingface/Huggingface_wandb.ipynb" >}}
-Visualize your [Hugging Face](https://github.com/huggingface/transformers) model's performance quickly with a seamless [W&B](https://wandb.ai/site) integration.
+Visualize your [Hugging Face](https://github.com/huggingface/transformers)
```

Comment:
Can you please undo the hard line wrapping that is introduced in this PR? Markdown is easier to maintain and review with soft wrapping per paragraph.